### ISSUE TITLE
Releases

### ISSUE THREAD

**Issue by hanthecat** [sanni/cartreader#292]
Hi, are the zips with the IDE and libraries supposed to be under releases? I only see them under the v3 hardware page.

https://github.com/sanni/cartreader/wiki/Cart-Reader-V3

**Comment by sanni** [comment]
You're absolutely right but the issue is that the old releases don't work with the new hardware V4, that'S why I have archived them to the Wiki and I currently can't make a new release for both V3/V4 because the file browser is crashing in the latest version because of low available memory. So I have to fix that first. Maybe this weekend. It's probably just an array that overflows.
